modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -47,12 +65,11 @@ G_TYPED_KERNEL(GStereo, <GMat(GMat, GMat, const StereoOutputFormat)>, "org.openc
 
 } // namespace calib3d
 
-/** @brief Extract disparity/depth information depending on passed StereoOutputFormat argument.
-The function extracts disparity/depth information depending on passed StereoOutputFormat argument from
-given stereo-pair.
+/// @brief Compute disparity/depth map for the specified stereo-pair.
```

Comment:
Why do use three slashes? M b i should too.

Reply:
/** instead

Reply:
Just a mistake 
FIxed